Source/ablastr/fields/PoissonSolver.H

```diff
@@ -106,7 +106,7 @@ computePhi (amrex::Vector<amrex::MultiFab*> const & rho,
             amrex::Vector<amrex::DistributionMapping> const& dmap,
             amrex::Vector<amrex::BoxArray> const& grids,
             T_BoundaryHandler const boundary_handler,
-            [[maybe_unused]] bool is_solver_multigrid,
+            [[maybe_unused]] bool is_solver_igf_on_lev0,
```

Comment:
This should now always be checked:
```suggestion
            bool is_solver_igf_on_lev0,
```